requirements.txt

```diff
@@ -15,5 +15,4 @@ panda-gym==1.1.1 # tmp fix: until compatibility with panda-gym v2
 rliable>=1.0.5
 wandb
 ale-py==0.7.4 # tmp fix: until new SB3 version is released
-# TODO: replace with release
-git+https://github.com/huggingface/huggingface_sb3
+huggingface_sb3>=2.2.1
```

Comment:
I would put a fix version here in case of breaking changes

Reply:
I assume huggingface_sb3 uses semantic versioning. Then we could just do `huggingface_sb3>=2.2.1, <3.*` right? What do you think?